services/WidgetsService.js

```diff
@@ -52,10 +52,10 @@ export default class WidgetsService {
     });
   }
 
-  fetchData({ id }) {
+  fetchData({ id, includes = '' }) {
     return new Promise((resolve, reject) => {
       get({
-        url: `${process.env.WRI_API_URL}/widget/${id}`,
+        url: `${process.env.WRI_API_URL}/widget/${id}?includes=${includes}`,
```

Comment:
We were talking about this. I think that it would be better if you handle with 'query-string'. There is an example here => https://github.com/resource-watch/resource-watch/blob/develop/components/dashboards/wysiwyg/widget-block-edition/widget-block-edition-actions.js#L21

Reply:
cool, thanks for mentioning that. I'll try to use from now on.
It's been included in this commit: a9aa17edc162db40a82405361b32d5b13815f987